### ISSUE TITLE
Compiler warning under Cygwin

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap#1140]
The version of Cygwin on AppVeyor is different from the one we use on Jenkins, and AppVeyor builds produce the following warning:
```
gcc -I. -DCONFIG_H   -Wall -g -O2  -g -O2  -o macfloat.o -c ../../src/macfloat.c
In file included from ../../src/macfloat.c:27:0:
../../src/macfloat.c: In function ‘FuncEXP10_MACFLOAT’:
../../src/macfloat.c:376:25: warning: implicit declaration of function ‘exp10’ [-Wimplicit-function-declaration]
 MAKEMATHPRIMITIVE(EXP10,exp10)
                         ^
../../src/macfloat.h:31:20: note: in definition of macro ‘MATH’
 #define MATH(name) name
                    ^
../../src/macfloat.c:376:1: note: in expansion of macro ‘MAKEMATHPRIMITIVE’
 MAKEMATHPRIMITIVE(EXP10,exp10)
 ^
../../src/macfloat.c:376:25: warning: incompatible implicit declaration of built-in function ‘exp10’
 MAKEMATHPRIMITIVE(EXP10,exp10)
                         ^
../../src/macfloat.h:31:20: note: in definition of macro ‘MATH’
 #define MATH(name) name
                    ^
../../src/macfloat.c:376:1: note: in expansion of macro ‘MAKEMATHPRIMITIVE’
 MAKEMATHPRIMITIVE(EXP10,exp10)
 ^
```

**Comment by fingolfin** [comment]
That warning is addressed by PR #1136

**Comment by fingolfin** [comment]
This has been fixed by the merge of PR #1136, specifically by commit d887b49a75d752a8f695504cdbfec105fa320cf9
